data_portal/models/labmetadata.py

```diff
@@ -85,11 +86,26 @@ class LabMetadataWorkflow(models.TextChoices):
     RESEARCH = "research"
 
 
+def remove_not_sequenced(qs: QuerySet) -> QuerySet:
```

Comment:
Nice! 

1 function-_nality_ added; 2 unit test cases follow suit. Make things robust a tad more. That's the spirit, yes! 👍